README.md

```diff
@@ -1,15 +1,55 @@
 ![Bitcoin-S logo](website/static/img/bitcoin-s-dark-logo.png)
 [![Build Status](https://github.com/bitcoin-s/bitcoin-s/workflows/Release/badge.svg)](https://github.com/bitcoin-s/bitcoin-s/actions) [![Coverage Status](https://coveralls.io/repos/github/bitcoin-s/bitcoin-s/badge.svg?branch=master)](https://coveralls.io/github/bitcoin-s/bitcoin-s?branch=master) [![Maven Central](https://img.shields.io/badge/Maven%20Central-0.6.0-brightgreen.svg)](https://mvnrepository.com/artifact/org.bitcoin-s) [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/bitcoin-s-core)
```

Comment:
Maybe these shields can be updated too? I wish they would automatically update, but the latest release is 1.7.0

Reply:
> Maybe these shields can be updated too? I wish they would automatically update, but the latest release is 1.7.0

Will do!